experimental/GModule/test/runtests.jl

```diff
@@ -44,6 +44,29 @@ end
   @test length(Oscar.RepPc.reps(GF(7, 6), G)) == 7
   @test length(Oscar.RepPc.reps(GF(2, 6), G)) == 3
 
+  G = SL(2,3)
+  F = GF(3,2)
+  L = [
+          matrix(F.([0 0 1; 1 0 0; 0 1 0])),
+          matrix(F.([2 0 0; 0 1 0; 0 0 2])),
+          matrix(F.([2 0 0; 0 2 0; 0 0 1])),
+          matrix(F.([1 0 0; 0 1 0; 0 0 1]))
```

Comment:
This seems weird, shouldn't it be this (and then similar a few lines later)? I mean both will work, but the current code allocates more.
```suggestion
          matrix(F, [0 0 1; 1 0 0; 0 1 0]),
          matrix(F, [2 0 0; 0 1 0; 0 0 2]),
          matrix(F, [2 0 0; 0 2 0; 0 0 1]),
          matrix(F, [1 0 0; 0 1 0; 0 0 1])
```